docs/src/main/asciidoc/spring-cloud-stream-support.adoc

```diff
@@ -74,19 +74,60 @@ Below options are used to configure Azure Event Hubs SDK Client.
-|*spring.cloud.azure.eventhubs*.processor.checkpointStore.* | NA | Blob checkpoint store configuration options.
-|*spring.cloud.azure.eventhubs*.processor.checkpointStore.createContainerIfNotExists | Boolean | If allowed to create container if not exists.
-|*spring.cloud.azure.eventhubs*.processor.checkpointStore.customerProvidedKey | String | Base64 encoded string of the encryption key.
-|*spring.cloud.azure.eventhubs*.processor.checkpointStore.encryptionScope | String | Encryption scope to encrypt blob contents on the server.
-|*spring.cloud.azure.eventhubs*.processor.checkpointStore.serviceVersion | BlobServiceVersion | The versions of Azure Storage Blob supported by this client library.
-|*spring.cloud.azure.eventhubs*.processor.checkpointStore.blobName | String | Storage blob name.
-|*spring.cloud.azure.eventhubs*.processor.checkpointStore.containerName | String | Storage container name.
+|*spring.cloud.azure.eventhubs*.domain-name | String | Domain name of an Azure Event Hubs
+Namespace value.
+|*spring.cloud.azure.eventhubs*.event-hub-name | String | Name of an Event Hub entity.
+|*spring.cloud.azure.eventhubs*.custom-endpoint-address | String | Custom Endpoint address.
+|*spring.cloud.azure.eventhubs*.is-shared-connection | Boolean | Whether to use the same
+connection for different Event Hub producer / consumer client.
+|*spring.cloud.azure.eventhubs*.processor.checkpoint-store.* | NA | Blob checkpoint store
+configuration options.
+|*spring.cloud.azure.eventhubs*.producer.* | NA | Producer
+configuration options.
+|*spring.cloud.azure.eventhubs*.consumer.* | NA | Consumer
+configuration options.
+|*spring.cloud.azure.eventhubs*.consumer.consumerGroup | NA | Name of the consumer group this consumer is associated with.
+|*spring.cloud.azure.eventhubs*.consumer.prefetchCount | NA | The number of events the Event Hub
+consumer will actively receive and queue locally without regard to whether a receiving operation is currently active.
+|*spring.cloud.azure.eventhubs*.processor.trackLastEnqueuedEventProperties | Boolean |Whether the event processor should request information on the last enqueued event on its associated partition, and track that information as events are received.
+|*spring.cloud.azure.eventhubs*.processor.initialPartitionEventPosition | Map |The map
+containing the event position to use for each partition if a checkpoint for the partition does not exist in CheckpointStore.
+not the event processor should request information on the last enqueued event on its associated partition, and track that information as events are received.
+|*spring.cloud.azure.eventhubs*.processor.partitionOwnershipExpirationInterval | Duration |The time duration after which the ownership of partition expires if it's not renewed by the owning processor instance.
+|*spring.cloud.azure.eventhubs*.processor.batch. maxSize| Integer | Max size of message in one
```

Comment:
We should follow the javadocs of eventhubs sdk about this property: The maximum number of events in a batch.

Reply:
updated